misc/scripts/update.sh

```diff
@@ -36,8 +36,11 @@ sudo chown "$PACSTALL_USER" -R /tmp/pacstall
 sudo mkdir -p /usr/share/bash-completion/completions
 
 STGDIR="/usr/share/pacstall"
+tabs -4
 
 tty_settings=$(stty -g)
+old_pacstall_version=( $(pacstall -V) )
+old_pacstall_update_info=( $(cat $STGDIR/repo/update) )
```

Comment:
I had requested you [here](https://github.com/pacstall/pacstall/pull/588#discussion_r884305527) to split the array into two variables. I want index `0` of the array to be it's own variable called `old_pacstall_update_username` and index `1` of the array to be in `old_pacstall_update_branch`.

Do the same for `new_pacstall_update_info`